tests/Unit/Domain/CoordinateMaps/Test_Wedge3D.cpp

```diff
@@ -0,0 +1,265 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <array>
+#include <catch.hpp>
+#include <random>
+
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/Wedge3D.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "tests/Unit/Domain/CoordinateMaps/TestMapHelpers.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge3D.Sphere",
```

Comment:
Why is this test called "Sphere"?  All the maps in it have sphericity=0.

Reply:
It tests points which only lie on the spherical surface as opposed to the general surface. The test doesn't depend on the value of the sphericity.